spec/models/user_spec.rb

```diff
@@ -417,7 +417,7 @@
     describe '#has_cached_role?' do
       describe 'when user has a role' do
         it 'returns true when the user has the role' do
-          user = create(:user, role_ids: organizer_role.id)
+          user = create(:organizer, resource: conference)
```

Comment:
what about using organizer (the one defined in line 13) instead of redefining the same again?

Reply:
Done in latest commit.